src/core/webidl.js

```diff
@@ -131,10 +132,17 @@ function createIdlAnchor(escaped, data, parent) {
   if (isDefaultJSON) {
     return hyperHTML`<a data-lt="default toJSON operation">${escaped}</a>`;
   }
+  const unlinkedAnchor = hyperHTML`<a
+    data-link-type="idl"
+    data-idl="${data.partial ? "partial" : null}"
+    data-title="${escaped}"
+    data-xref-type="${getDfnType(data.type)}">${escaped}</a>`;
+
+  // Partial interfaces must always link somewhere else.
+  if (data.partial && !dfn) {
```

Comment:
Might be able to merge this with `showWarnings` below.

Reply:
I spent like an hour trying to... and I couldn't do it 😭 There are subtle differences in how they work and in the end I couldn't figure it out. 

We can maybe try again as a follow up, but I'd like to get this out as a bunch of specs are currently broken.